src/serializer.rs

```diff
@@ -90,7 +75,7 @@ impl<'a> ToCss for Token<'a> {
                 serialize_unquoted_url(&**value, dest)?;
                 dest.write_str(")")?;
             },
-            Token::Delim(value) => write!(dest, "{}", value)?,
+            Token::Delim(value) => dest.write_str(value.encode_utf8(&mut [0_u8; 4]))?,
```

Comment:
Can't this use `write_char`?

Reply:
Not sure why I didn’t think of that… Done :)